src/cascadia/TerminalSettingsModel/Command.idl

```diff
@@ -36,6 +36,7 @@ namespace Microsoft.Terminal.Settings.Model
         Command();
 
         String Name { get; };
+        String ID;

```

Comment:
FYI this allows somebody who has a Command object to _set_ the ID... but none of our other properties have setters!